src/geom/GeometryCollection.cpp

```diff
@@ -39,6 +39,7 @@ GeometryCollection::GeometryCollection(const GeometryCollection& gc)
     :
     Geometry(gc),
     geometries(gc.geometries.size()),
+    flags{}, // set all flags to zero
```

Comment:
This is a copy constructor. Why not copying the flags as well ?
```suggestion
    flags(gc.flags),
```